src/auth/auth.service.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ForbiddenException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { UserService } from 'src/user/user.service';
+import { JwtService } from '@nestjs/jwt';
+import { User } from '@prisma/client';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private userService: UserService,
+    private jwtService: JwtService,
+  ) {}
+
+  async signIn(email: string, pass: string): Promise<any> {
```

Comment:
from what I see we are returning `Promise<{ access_token: string }>` here. Replace `any` with a proper return type. Also you can declare the type (or interface) either in a separate file or somewhere in this service. I'd vote for a separate file